### PR TITLE
Update README adding information

### LINKED ISSUE #3: Update README
- Full description of the application
- Short video demo of the app in action
- Download buttons for Play Store and App Store
- Description of what Ditto is
- Link to documentation in Ditto
- Discussion of why this app is important -- demonstration of counters

### PR DESCRIPTION
**Pull request by kndoshn**
Closes #3

### CODE REVIEW

**Comment by okdistribute** [comment]
@rratner Can you review?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,22 @@
-## Inventory
+# Inventory
+
+Demo app for showing how Ditto sync and counters work.
```
Comment (rratner) [inline README.md:3]: I would say "Demo app for showcasing Ditto's real-time sync and Conflict Resolution through the use of an inventory counter."

File: README.md [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+
+This inventory demo showcases the smoothness of the Ditto sync and how counters work with Ditto. You can open up the Presence Viewer to see the Ditto connections too.
```
Comment (rratner) [inline README.md:5]: "the smoothness of Ditto's sync and conflict resolution, and how counters work with Ditto. You can also open up the Presence Viewer to see all existing devices and connections in the mesh"

File: README.md [hunk 3]
```diff
@@ -1,4 +1,22 @@
...
+
+- [Demo Video](https://www.youtube.com/watch?v=1P2bKEJjdec)
+
+- [iOS Download](https://apps.apple.com/us/app/ditto-inventory/id1449905935)
+- [Android Download](https://play.google.com/store/apps/details?id=live.ditto.inventory)
+
+
+## Developing
+
+If you are a member of Ditto, please access to this page and copy-paste the environment variables to this project in order to build the app:
```
Comment (rratner) [inline README.md:17]: "If you work for Ditto, please visit this page..."

Comment (okdistribute) [reply README.md:17]: We should move this out into a Notion page to reduce noise

**Pull request merged**